src/TMedia.cpp

```diff
@@ -687,7 +768,7 @@ TMediaPlayer TMedia::matchMediaPlayer(TMediaData& mediaData, const QString& abso
 #endif
                 pPlayer = pTestPlayer;
                 pPlayer.setMediaData(mediaData);
-                pPlayer.getMediaPlayer()->setVolume(mediaData.getMediaVolume());
+                pPlayer.getMediaPlayer()->setVolume(mediaData.getMediaFadeIn() != TMediaData::MediaFadeNotSet ? 1 : mediaData.getMediaVolume());
```

Comment:
In the previous change (added a warning about being `nable to create new QMediaPlayer object.`) there is a comment that says `// It should be impossible to ever reach this.` - but if it is **not** impossible then this line at least, will cause a seg. fault. So, should it handle it or not?

Reply:
Good to check on, thank you! I simulated this and got the warning to the console and continued operations of Mudlet.

<img width="744" alt="Screen Shot 2022-01-09 at 5 01 40 PM" src="https://user-images.githubusercontent.com/3058178/148702841-16cbc9d3-5eed-41ff-9fee-15df7c501873.png">